SpatialGDK/Source/SpatialGDKEditorToolbar/Public/SpatialGDKEditorToolbar.h

```diff
@@ -191,4 +197,8 @@ class FSpatialGDKEditorToolbarModule : public IModuleInterface, public FTickable
 	bool bStartingCloudDeployment;
 
 	void GenerateConfigFromCurrentMap();
+
+	// Used to show worker boundaries in the editor
+	ASpatialDebuggerEditor* SpatialDebuggerEditor;
```

Comment:
should this be a UPROPERTY or weak pointer of some kind?

Reply:
Added UPROPERTY